frontend/src/CroppedImage.js

```diff
@@ -1,19 +1,31 @@
 import ReactCrop from 'react-image-crop'
 import 'react-image-crop/dist/ReactCrop.css'
 
-function CroppedImage({src, alt}) {
+function CroppedImage({src, alt, title}) {
+
+  const image = <img {...{src, alt}} />
+  const caption = <figcaption>{title}</figcaption>
+  
   let fragment = src.match(/#xywh=percent:([.\d]+),([.\d]+),([.\d]+),([.\d]+)/);
   if (fragment) {
     let [_, x, y, width, height] = fragment;
     let crop = {unit: '%', x, y, width, height};
     return (
-      <ReactCrop {...{crop}} disabled={true} >
-        <img {...{src, alt}} />
-      </ReactCrop>
+      <div>
+        <ReactCrop {...{crop}} disabled={true} >
+          {renderImg(src, alt)}
+        </ReactCrop>
+        {renderCaption(title)}
```

Comment:
Please note that HTML5 `<figcaption>` should be in a `<figure>`.

Here is a documentation to the Bootstrap classes you should use: https://getbootstrap.com/docs/5.0/content/figures/